containers/jenkins-slave-vivid/config/scripts/postStart.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+ORIGIN_DIR=/var/jenkins_home
+TARGET_DIR=/home/jenkins-slave
+cp $ORIGIN_DIR/ssh-key/id-rsa $TARGET_DIR/.ssh/id_rsa
+cp $ORIGIN_DIR/ssh-key/id-rsa.pub $TARGET_DIR/.ssh/id_rsa.pub
```

Comment:
I tried to put a symlink there but udf didn't like it... We are using here `ssh-key/id-rsa*` because of kubernetes secret names' restrictions, if we could pass udf the name of the keys to be used it would be great.